react.code-workspace

```diff
@@ -0,0 +1,28 @@
+{
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "extensions": {
+    "recommendations": [
+      "dbaeumer.vscode-eslint",
+      "editorconfig.editorconfig",
+      "esbenp.prettier-vscode",
+      "flowtype.flow-for-vscode"
+    ]
+  },
+  "settings": {
+    "editor.formatOnSave": true,
+    "flow.pathToFlow": "${workspaceFolder}/node_modules/.bin/flow",
+    "javascript.validate.enable": false,
+    "search.useIgnoreFiles": true,
+    "search.exclude": {
+      "**/dist/**": true,
+      "**/build/**": true,
+      "**/out/**": true,
+      "*.map": true,
+      "*.log": true
```

Comment:
* I just checked and `search.useIgnoreFiles` can be removed indeed.
* `search.exclude` is an important one because it enables search with "include path" to ignore these paths. VSCode doesn't respect `.gitignore` when "include path" is used.
![image](https://github.com/facebook/react/assets/5188459/92a07473-4dee-4f24-871e-563fb291a100)

Reply:
ah gotcha